presto-main/src/main/java/com/facebook/presto/server/ResourceGroupStateInfo.java

```diff
@@ -94,6 +102,18 @@ public DataSize getMemoryUsage()
     }
 
     @JsonProperty
+    public int getNumSubGroups()
+    {
+        return numSubGroups;
+    }
+
+    @JsonProperty
+    public List<ResourceGroupId> getEligibleSubGroups()
```

Comment:
Eligible sub groups are the subgroups that this resource group would consider when scheduling (meaning that they have a can_run status). So when you think about subgroups that are competing for resource, only eligible sub groups should be considered.